net/ser2net/files/etc/config/ser2net

```diff
@@ -0,0 +1,7 @@
+config 'proxy'
```

Comment:
Hm, "proxy" is not my first choice, but in the absence of a better idea... The important thing is, that we might add additional types later, e.g. for LED configuration etc.

Reply:
Why not just device?  one section for each device line?

Reply:
there is already a parameter device? I think changing this device makes it confusing. And I suppose ser2net is proxying the serial data to net. So it seems ok to me?

Reply:
The port # and the device name are unique strings...  I would differentiate the blocks based on either one of those, and make the other one a mandatory parameter.

Reply:
well, they're not actually. ser2net can have the same port and multiple devices, or multiple ports for the same device, and have them enabled/disabled separately at runtime via the control port.